org.fsf.pspp.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.fsf.pspp
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: psppire
+modules:
+  - name: spread-sheet-widget
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://alpha.gnu.org/gnu/ssw/spread-sheet-widget-0.8.tar.gz
+        sha256: 8589d8298fcf3b5850d0968b04801a4f40faf0555544f6cc9d954b0162e9954b
+  - name: gsl
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.7.1.tar.gz
+        sha256: dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b
+  - name: pspp
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/pspp/pspp-1.6.0.tar.gz
+        sha256: 8d5b51dba8ad812ee93288ad664d8184f164d3b394d2f9c09fcb39f86e0b0a12
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --own-name=gnu.pspp
```

Comment:
I have questions about the validity of that D-Bus name. Note that this is not blocking here assuming this is what upstream uses.

Reply:
I think this is supposed to match the parameter to gtk_application_new(), which uses `gnu.pspp` as shown here: https://git.savannah.gnu.org/cgit/pspp.git/tree/src/ui/gui/main.c#n525

Reply:
I know, but that remain there are questions since `gnu` isn't a valid tld. 

https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names-bus

Reply:
and just to add, by default we grant the `app-id` valid as the `own-name`, hence the idea that they should match.